packages/acceptance-tests/scripts/generate-expected-file-list.test.js

```diff
@@ -6,8 +6,6 @@ const expected = [
   "has-updates-css-2.css",
   "has-updates-css-1.css",
   "has-updates-css-0.css",
-  "invalid-css-file-0.css",
```

Comment:
Are we no expecting these 2 files anymore? Is this still not a valid test to check for invalid CSS?